styles/scss/core/core.scss

```diff
@@ -16,6 +16,7 @@
 // Paragon core CSS
 @import "../../css/core/variables";
 @import "../../css/core/custom-media-breakpoints";
+@import "../../css/core/abstract-variables";
```

Comment:
Looks like the change mentioned [here](https://github.com/openedx/paragon/pull/3222#discussion_r1770577421) didn't make it in?

Reply:
Corrected to `abstraction-variables`

Reply:
This comment was about the import order, not the naming